.github/workflows/pr_tests.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   main:
+    name: Test Checks
```

Comment:
Will be tested in dev branch. I try to enable required PR checks before merge. For now it doesn't work, because list of available checks in settings is empty